fast/stages/1-resman/stage-2-networking.tf

```diff
@@ -60,35 +60,25 @@ module "net-folder" {
     # network security stage 2 service accounts
     var.fast_stage_2.network_security.enabled != true ? {} : {
       "roles/serviceusage.serviceUsageAdmin" = [
-        try(module.nsec-sa-rw[0].iam_email, null)
+        module.nsec-sa-rw[0].iam_email
       ]
       (var.custom_roles["network_firewall_policies_admin"]) = [
-        try(module.nsec-sa-rw[0].iam_email, null)
+        module.nsec-sa-rw[0].iam_email
       ]
       "roles/compute.orgFirewallPolicyUser" = [
-        try(module.nsec-sa-ro[0].iam_email, null)
+        module.nsec-sa-ro[0].iam_email
       ]
       "roles/serviceusage.serviceUsageConsumer" = [
-        try(module.nsec-sa-ro[0].iam_email, null)
+        module.nsec-sa-ro[0].iam_email
       ]
     },
     # security stage 2 service accounts
     var.fast_stage_2.security.enabled != true ? {} : {
       "roles/serviceusage.serviceUsageAdmin" = [
-        try(module.sec-sa-rw[0].iam_email, null)
+        module.sec-sa-rw[0].iam_email
       ]
       "roles/serviceusage.serviceUsageConsumer" = [
-        try(module.sec-sa-ro[0].iam_email, null)
-      ]
-    },
-    try(var.custom_roles["network_firewall_policies_admin"], null) == null ? {} : {
```

Comment:
as mentioned above, these are needed until we refactor the nsec stage and keep fw policies in the net stages, they are currently managed from nsec and attached to VPCs here

Reply:
Agreed, but the one I removed here is granting to the **security SA** (not nsec SA). 

The correct binding (to the nsec SA) is already done a few lines above together with all the other nsec bindings